tenets/codelingo/code-review-comments/use-crypto-rand/expected.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+   "Comment": "Do not use package math/rand to generate keys, even \nthrowaway ones. Unseeded, the generator is completely predictable. \nSeeded with time.Nanoseconds(), there are just a few bits of entropy. \nInstead, use crypto/rand's Reader, and if you need text, print to \nhexadecimal or base64\n",
+   "Filename": "effective-go/use-crypto-rand/example-bad.go",
+   "Line": 4,
```

Comment:
attach the comment to where it's being used to gen the key. There should be two results here.